modules/distr-tracing-tempo-config-storage.adoc

```diff
@@ -0,0 +1,34 @@
+// Module included in the following assemblies:
+//
+// * distr_tracing_tempo/distr-tracing-tempo-configuring.adoc
+
+:_content-type: REFERENCE
+[id="distr-tracing-tempo-config-storage_{context}"]
+= Distributed tracing storage configuration options
+
+You configure storage for the Collector, Ingester, Distributor, and Query services under `spec.storage`. Multiple instances of each of these components can be provisioned as required for performance and resilience purposes.
+
+.General storage parameters used by the {TempoOperator} to define distributed tracing storage
+[options="header"]
+[cols="l, a, a, a"]
+|===
+|Parameter |Description |Values |Default value
+|spec:
+  storage:
+    type:
+|Type of storage to use for the deployment.
+|`memory`. Memory storage is only appropriate for development, testing, demonstrations, and proof of concept environments because the data does not persist when the pod is shut down.
+|`memory`
+
+|storage:
+  secretname:
+|Name of the secret, for example `tracing-secret`.
+|
+|N/A
+
+|storage:
+  options: {}
+|Configuration options that define the storage.
+|
+|
+|===
```

Comment:
```suggestion
For each type of storate we need to create a secret with differnt properties, here are some examples:

kubectl create secret generic tempostack-dev-s3 \
  --from-literal=bucket="<BUCKET_NAME>" \
  --from-literal=endpoint="<AWS_BUCKET_ENDPOINT>" \
  --from-literal=access_key_id="<AWS_ACCESS_KEY_ID>" \
  --from-literal=access_key_secret="<AWS_ACCESS_KEY_SECRET>"


Azure

`kubectl create secret generic tempostack-dev-azure \
  --from-literal=container="<AZURE_CONTAINER_NAME>" \
  --from-literal=account_name="<AZURE_ACCOUNT_NAME>" \
  --from-literal=account_key="<AZURE_ACCOUNT_KEY>"
`

Google cloud

`kubectl create secret generic tempostack-dev-gcs \
  --from-literal=bucketname="<BUCKET_NAME>" \
  --from-file=key.json="<PATH/TO/KEY.JSON>"
`

Reply:
@rubenvp8510, I can include here the Table 1 from https://62410--docspreview.netlify.app/openshift-enterprise/latest/distr_tracing/distr_tracing_tempo/distr-tracing-tempo-installing#distr-tracing-tempo-install-web-console_dist-tracing-tempo-installing. YES/NO?

Reply:
Absolutely YES!